src/textField/index.tsx

```diff
@@ -63,18 +65,23 @@ export interface TextFieldState {
   readonly showPassword: boolean
 }
 
-const DisplayError = (error: string | JSX.Element) => {
+const DisplayError = (error: errorField) => {
   const className = 'kirk-error-message'
   if (React.isValidElement(error)) {
     return React.cloneElement(error as React.ReactElement<any>, {
       className,
     })
   }
-  return (
-    <span role="alert" className={className}>
-      {error}
-    </span>
-  )
+
+  if (typeof error !== 'boolean') {
```

Comment:
So, nothing is done if it's a boolean?

Reply:
Good point, I'm going to remove this.

Edit: Wow, I just moved back to check the implementation of this component, and Error used to be a boolean to put a red background, not the error message, and then the behavior changed.. I'm gonna update this